src/Compiler/Checking/PostInferenceChecks.fs

```diff
@@ -2597,6 +2597,7 @@ and CheckModuleSpec cenv env mbind =
 let CheckImplFileContents cenv env implFileTy implFileContents  = 
     let rpi, mhi = ComputeRemappingFromImplementationToSignature cenv.g implFileContents implFileTy
     let env = { env with sigToImplRemapInfo = (mkRepackageRemapping rpi, mhi) :: env.sigToImplRemapInfo }
+    UpdatePrettyTyparNames.updateModuleOrNamespaceType implFileTy
```

Comment:
If we call this for a .fs impl that had a backing .fsi signature file, it will be just a no-op since the name prettyification already happened, right?